### PR TITLE
better vignette index in pkgdown?

### PR DESCRIPTION
**Pull request by maelle**
* All vignette titles now start with a capital letter and are a sentence
* Add an order from easy/most basic usage to advanced.

### CODE REVIEW

File: vignettes/configuration.Rmd [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "vcr configuration"
+title: "Tweak vcr default configuration"
```
Comment (sckott) [inline vignettes/configuration.Rmd:2]: Could we use a different word than tweak? would "manage vcr configuration" be better?

File: vignettes/request_matching.Rmd [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "request matching"
+title: "Tweak vcr request matching"
```
Comment (sckott) [inline vignettes/request_matching.Rmd:2]: similar comment, could we maybe not use tweak

**Review by sckott — CHANGES_REQUESTED** [review]
In the vignette title "Why and how edit your vcr cassettes?" could we change to "Why and how __to__ edit your vcr cassettes?"

**Comment by maelle** [comment]
I made changes. Now I wonder whether it'd be clearer if request matching were merged into "configure vcr" as it's part of configuration?

**Comment by sckott** [comment]
No, I think request matching just deserves more attention than most other configuration settings - we should at some point fill out that matching vignette more

**Review by sckott — APPROVED** [review]

**Pull request merged**